src/sentry/incidents/utils/types.py

```diff
@@ -10,13 +10,6 @@ class QuerySubscriptionUpdate(TypedDict):
     timestamp: datetime
 
 
-class MetricDetectorUpdate(TypedDict):
```

Comment:
sorry i snuck this into the pr, i noticed it's redundant with `QuerySubscriptionUpdate` and updated the use.